packages/ember-views/lib/views/view.js

```diff
@@ -994,6 +994,58 @@ var View = CoreView.extend(
   },
 
   /**
+    @private
+
+    Creates a new DOM element, renders the view into it, then returns the
+    element.
+
+    By default, the element created and rendered into will be a `BODY` element,
+    since this is the default context that views are rendered into when being
+    inserted directly into the DOM.
+
+    ```js
+    var element = view.renderToElement();
+    element.tagName; // => "BODY"
+    ```
+
+    You can override the kind of element rendered into and returned by
+    specifying an optional tag name as the first argument.
+
+    ```js
+    var element = view.renderToElement('table');
+    element.tagName; // => "TABLE"
+    ```
+
+    This method is useful if you want to render the view into an element that
+    is not in the document's body. Instead, a new `body` element, detached from
+    the DOM is returned. FastBoot uses this to serialize the rendered view into
+    a string for transmission over the network.
+
+    ```js
+    app.visit('/').then(function(instance) {
+      var element;
+      Ember.run(function() {
+        element = renderToElement(instance);
+      });
+
+      res.send(serialize(element));
+    });
+    ```
+
+    @method renderToElement
+    @param {String} tagName The tag of the element to create and render into. Defaults to "body".
+    @return {HTMLBodyElement} element
+  */
+  renderToElement: function(tagName) {
+    tagName = tagName || 'body';
+
+    var element = this.renderer._dom.createElement(tagName);
```

Comment:
I would prefer to pass the created element in, fwiw. The view is not supposed to access `_dom` directly, and making this a public API would couple us to always exposing a dom helper to views.

In that case, why not just bypass the renderer or make the `dom` property not underscore.